items:
- uid: com.microsoft.azure.batch._create_tasks_terminated_exception
  id: _create_tasks_terminated_exception
  parent: com.microsoft.azure.batch
  children:
  - classcom.microsoft.azure.batch._create_tasks_terminated_exception.CreateTasksTerminatedException(final String,List< TaskAddResult >,List< TaskAddParameter >)
  - com.microsoft.azure.batch._create_tasks_terminated_exception.failureTasks()
  - com.microsoft.azure.batch._create_tasks_terminated_exception.pendingTaskList()
  href: com.microsoft.azure.batch._create_tasks_terminated_exception.yml
  langs:
  - java
  name: CreateTasksTerminatedException
  fullName: com.microsoft.azure.batch.CreateTasksTerminatedException
  type: Class
  source:
    remote: &o0
      path: azure-batch/src/main/java/com/microsoft/azure/batch/CreateTasksTerminatedException.java
      branch: master
      repo: https://github.com/Azure/azure-sdk-for-java.git
    path: azure-batch/src/main/java/com/microsoft/azure/batch/CreateTasksTerminatedException.java
    startLine: 17
  package: com.microsoft.azure.batch
  summary: >-
    <p>

      <xref href="com.microsoft.azure.batch._create_tasks_terminated_exception" data-throw-if-not-resolved="false"></xref>. </p>
  syntax: &o1
    content: public class CreateTasksTerminatedException extends BatchErrorException
  inheritance:
  - java.lang.Object
  - Exception
  - com.microsoft.rest._rest_exception
  - com.microsoft.azure.batch.protocol.models._batch_error_exception
  inheritedMembers:
  - com.microsoft.azure.batch.protocol.models._batch_error_exception.BatchErrorException()
  - classcom.microsoft.azure.batch.protocol.models._batch_error_exception.BatchErrorException(final String)
  - classcom.microsoft.azure.batch.protocol.models._batch_error_exception.BatchErrorException(final String,final Throwable)
  - classcom.microsoft.azure.batch.protocol.models._batch_error_exception.BatchErrorException(final Throwable)
  - com.microsoft.azure.batch.protocol.models._batch_error_exception.getBody()
  - com.microsoft.azure.batch.protocol.models._batch_error_exception.getResponse()
  - com.microsoft.rest._rest_exception.RestException()
  - com.microsoft.rest._rest_exception.RestException(String)
  - com.microsoft.rest._rest_exception.RestException(Throwable)
  - com.microsoft.rest._rest_exception.RestException(String,Throwable)
  - com.microsoft.azure.batch.protocol.models._batch_error_exception.setBody(BatchError)
  - com.microsoft.azure.batch.protocol.models._batch_error_exception.setResponse(Response)
- uid: classcom.microsoft.azure.batch._create_tasks_terminated_exception.CreateTasksTerminatedException(final String,List< TaskAddResult >,List< TaskAddParameter >)
  id: CreateTasksTerminatedException(final String,List< TaskAddResult >,List< TaskAddParameter >)
  parent: com.microsoft.azure.batch._create_tasks_terminated_exception
  href: com.microsoft.azure.batch._create_tasks_terminated_exception.yml
  langs:
  - java
  name: CreateTasksTerminatedException(final String message, List< TaskAddResult > failureTasks, List< TaskAddParameter > pendingList)
  fullName: com.microsoft.azure.batch.CreateTasksTerminatedException.CreateTasksTerminatedException(final String message, List< TaskAddResult > failureTasks, List< TaskAddParameter > pendingList)
  type: Constructor
  source:
    remote: *o0
    path: azure-batch/src/main/java/com/microsoft/azure/batch/CreateTasksTerminatedException.java
    startLine: 26
  package: com.microsoft.azure.batch
  summary: >-
    <p>Initializes a new instance of the <xref href="com.microsoft.azure.batch._create_tasks_terminated_exception" data-throw-if-not-resolved="false"></xref> class.</p>

    <p></p>
  syntax:
    content: public CreateTasksTerminatedException(final String message, List< TaskAddResult > failureTasks, List< TaskAddParameter > pendingList)
    parameters:
    - id: message
      type: 3d93591e
      description: <p>the exception message </p>
    - id: failureTasks
      type: f6a6204dcom.microsoft.azure.batch.protocol.models._task_add_result9ec3e006
      description: <p>the list of failed tasks </p>
    - id: pendingList
      type: f6a6204dcom.microsoft.azure.batch.protocol.models._task_add_parameter9ec3e006
      description: <p>the list of pending tasks </p>
- uid: com.microsoft.azure.batch._create_tasks_terminated_exception.failureTasks()
  id: failureTasks()
  parent: com.microsoft.azure.batch._create_tasks_terminated_exception
  href: com.microsoft.azure.batch._create_tasks_terminated_exception.yml
  langs:
  - java
  name: failureTasks()
  fullName: List<TaskAddResult> com.microsoft.azure.batch.CreateTasksTerminatedException.failureTasks()
  type: Method
  source:
    remote: *o0
    path: azure-batch/src/main/java/com/microsoft/azure/batch/CreateTasksTerminatedException.java
    startLine: 41
  package: com.microsoft.azure.batch
  syntax:
    content: public List< TaskAddResult > failureTasks()
    return:
      type: f6a6204dcom.microsoft.azure.batch.protocol.models._task_add_result9ec3e006
      description: <p>the list of failed tasks </p>
- uid: com.microsoft.azure.batch._create_tasks_terminated_exception.pendingTaskList()
  id: pendingTaskList()
  parent: com.microsoft.azure.batch._create_tasks_terminated_exception
  href: com.microsoft.azure.batch._create_tasks_terminated_exception.yml
  langs:
  - java
  name: pendingTaskList()
  fullName: List<TaskAddParameter> com.microsoft.azure.batch.CreateTasksTerminatedException.pendingTaskList()
  type: Method
  source:
    remote: *o0
    path: azure-batch/src/main/java/com/microsoft/azure/batch/CreateTasksTerminatedException.java
    startLine: 48
  package: com.microsoft.azure.batch
  syntax:
    content: public List< TaskAddParameter > pendingTaskList()
    return:
      type: f6a6204dcom.microsoft.azure.batch.protocol.models._task_add_parameter9ec3e006
      description: <p>the list of pending tasks </p>
references:
- uid: 3d93591e
  spec.java:
  - name: final String
    fullName: final String
- uid: f6a6204dcom.microsoft.azure.batch.protocol.models._task_add_result9ec3e006
  spec.java:
  - name: 'List< '
    fullName: 'List< '
  - uid: com.microsoft.azure.batch.protocol.models._task_add_result
    name: TaskAddResult
    fullName: com.microsoft.azure.batch.protocol.models.TaskAddResult
    href: com.microsoft.azure.batch.protocol.models._task_add_result.yml
  - name: " >"
    fullName: " >"
- uid: f6a6204dcom.microsoft.azure.batch.protocol.models._task_add_parameter9ec3e006
  spec.java:
  - name: 'List< '
    fullName: 'List< '
  - uid: com.microsoft.azure.batch.protocol.models._task_add_parameter
    name: TaskAddParameter
    fullName: com.microsoft.azure.batch.protocol.models.TaskAddParameter
    href: com.microsoft.azure.batch.protocol.models._task_add_parameter.yml
  - name: " >"
    fullName: " >"
- uid: com.microsoft.azure.batch.protocol.models._batch_error_exception.BatchErrorException()
  parent: com.microsoft.azure.batch.protocol.models._batch_error_exception
  href: com.microsoft.azure.batch.protocol.models._batch_error_exception.yml
  name: BatchErrorException()
  fullName: com.microsoft.azure.batch.protocol.models.BatchErrorException.BatchErrorException()
  type: Constructor
  summary: <p>Initializes a new instance of the <xref href="com.microsoft.azure.batch.protocol.models._batch_error_exception" data-throw-if-not-resolved="false"></xref> class. </p>
  syntax:
    content: public BatchErrorException()
- uid: classcom.microsoft.azure.batch.protocol.models._batch_error_exception.BatchErrorException(final String)
  parent: com.microsoft.azure.batch.protocol.models._batch_error_exception
  href: com.microsoft.azure.batch.protocol.models._batch_error_exception.yml
  name: BatchErrorException(final String message)
  fullName: com.microsoft.azure.batch.protocol.models.BatchErrorException.BatchErrorException(final String message)
  type: Constructor
  summary: >-
    <p>Initializes a new instance of the <xref href="com.microsoft.azure.batch.protocol.models._batch_error_exception" data-throw-if-not-resolved="false"></xref> class.</p>

    <p></p>
  syntax:
    content: public BatchErrorException(final String message)
    parameters:
    - id: message
      type: 3d93591e
      description: <p>The exception message. </p>
- uid: classcom.microsoft.azure.batch.protocol.models._batch_error_exception.BatchErrorException(final String,final Throwable)
  parent: com.microsoft.azure.batch.protocol.models._batch_error_exception
  href: com.microsoft.azure.batch.protocol.models._batch_error_exception.yml
  name: BatchErrorException(final String message, final Throwable cause)
  fullName: com.microsoft.azure.batch.protocol.models.BatchErrorException.BatchErrorException(final String message, final Throwable cause)
  type: Constructor
  summary: >-
    <p>Initializes a new instance of the <xref href="com.microsoft.azure.batch.protocol.models._batch_error_exception" data-throw-if-not-resolved="false"></xref> class.</p>

    <p></p>
  syntax:
    content: public BatchErrorException(final String message, final Throwable cause)
    parameters:
    - id: message
      type: 3d93591e
      description: <p>the exception message </p>
    - id: cause
      type: fa00b19f
      description: <p>exception that caused this exception to occur </p>
- uid: classcom.microsoft.azure.batch.protocol.models._batch_error_exception.BatchErrorException(final Throwable)
  parent: com.microsoft.azure.batch.protocol.models._batch_error_exception
  href: com.microsoft.azure.batch.protocol.models._batch_error_exception.yml
  name: BatchErrorException(final Throwable cause)
  fullName: com.microsoft.azure.batch.protocol.models.BatchErrorException.BatchErrorException(final Throwable cause)
  type: Constructor
  summary: >-
    <p>Initializes a new instance of the <xref href="com.microsoft.azure.batch.protocol.models._batch_error_exception" data-throw-if-not-resolved="false"></xref> class.</p>

    <p></p>
  syntax:
    content: public BatchErrorException(final Throwable cause)
    parameters:
    - id: cause
      type: fa00b19f
      description: <p>exception that caused this exception to occur </p>
- uid: com.microsoft.azure.batch.protocol.models._batch_error_exception.getBody()
  parent: com.microsoft.azure.batch.protocol.models._batch_error_exception
  href: com.microsoft.azure.batch.protocol.models._batch_error_exception.yml
  name: getBody()
  fullName: BatchError com.microsoft.azure.batch.protocol.models.BatchErrorException.getBody()
  type: Method
  summary: >-
    <p>Gets the HTTP response body.</p>

    <p></p>
  syntax:
    content: public BatchError getBody()
    return:
      type: com.microsoft.azure.batch.protocol.models._batch_error
      description: <p>the response body </p>
- uid: com.microsoft.azure.batch.protocol.models._batch_error_exception.getResponse()
  parent: com.microsoft.azure.batch.protocol.models._batch_error_exception
  href: com.microsoft.azure.batch.protocol.models._batch_error_exception.yml
  name: getResponse()
  fullName: Response com.microsoft.azure.batch.protocol.models.BatchErrorException.getResponse()
  type: Method
  summary: >-
    <p>Gets information about the associated HTTP response.</p>

    <p></p>
  syntax:
    content: public Response getResponse()
    return:
      type: e9d34ed6
      description: <p>the HTTP response </p>
- uid: com.microsoft.rest._rest_exception.RestException()
  parent: com.microsoft.rest._rest_exception
  href: com.microsoft.rest._rest_exception.yml
  name: RestException()
  fullName: com.microsoft.rest.RestException.RestException()
  type: Constructor
  summary: <p>Initializes a new instance of the AutoRestException class. </p>
  syntax:
    content: public RestException()
- uid: com.microsoft.rest._rest_exception.RestException(String)
  parent: com.microsoft.rest._rest_exception
  href: com.microsoft.rest._rest_exception.yml
  name: RestException(String message)
  fullName: com.microsoft.rest.RestException.RestException(String message)
  type: Constructor
  summary: >-
    <p>Initializes a new instance of the AutoRestException class.</p>

    <p></p>
  syntax:
    content: public RestException(String message)
    parameters:
    - id: message
      type: "26831127"
      description: <p>The exception message. </p>
- uid: com.microsoft.rest._rest_exception.RestException(Throwable)
  parent: com.microsoft.rest._rest_exception
  href: com.microsoft.rest._rest_exception.yml
  name: RestException(Throwable cause)
  fullName: com.microsoft.rest.RestException.RestException(Throwable cause)
  type: Constructor
  summary: >-
    <p>Initializes a new instance of the AutoRestException class.</p>

    <p></p>
  syntax:
    content: public RestException(Throwable cause)
    parameters:
    - id: cause
      type: a04fafd7
      description: <p>exception that caused this exception to occur </p>
- uid: com.microsoft.rest._rest_exception.RestException(String,Throwable)
  parent: com.microsoft.rest._rest_exception
  href: com.microsoft.rest._rest_exception.yml
  name: RestException(String message, Throwable cause)
  fullName: com.microsoft.rest.RestException.RestException(String message, Throwable cause)
  type: Constructor
  summary: >-
    <p>Initializes a new instance of the AutoRestException class.</p>

    <p></p>
  syntax:
    content: public RestException(String message, Throwable cause)
    parameters:
    - id: message
      type: "26831127"
      description: <p>the exception message </p>
    - id: cause
      type: a04fafd7
      description: <p>exception that caused this exception to occur </p>
- uid: com.microsoft.azure.batch.protocol.models._batch_error_exception.setBody(BatchError)
  parent: com.microsoft.azure.batch.protocol.models._batch_error_exception
  href: com.microsoft.azure.batch.protocol.models._batch_error_exception.yml
  name: setBody(BatchError body)
  fullName: void com.microsoft.azure.batch.protocol.models.BatchErrorException.setBody(BatchError body)
  type: Method
  summary: >-
    <p>Sets the HTTP response body.</p>

    <p></p>
  syntax:
    content: public void setBody(BatchError body)
    parameters:
    - id: body
      type: com.microsoft.azure.batch.protocol.models._batch_error
      description: <p>the response body </p>
- uid: com.microsoft.azure.batch.protocol.models._batch_error_exception.setResponse(Response)
  parent: com.microsoft.azure.batch.protocol.models._batch_error_exception
  href: com.microsoft.azure.batch.protocol.models._batch_error_exception.yml
  name: setResponse(Response response)
  fullName: void com.microsoft.azure.batch.protocol.models.BatchErrorException.setResponse(Response response)
  type: Method
  summary: >-
    <p>Sets the HTTP response.</p>

    <p></p>
  syntax:
    content: public void setResponse(Response response)
    parameters:
    - id: response
      type: e9d34ed6
      description: <p>the HTTP response </p>
- uid: com.microsoft.azure.batch.protocol.models._batch_error_exception
  parent: com.microsoft.azure.batch.protocol.models
  href: com.microsoft.azure.batch.protocol.models._batch_error_exception.yml
  name: BatchErrorException
  fullName: com.microsoft.azure.batch.protocol.models.BatchErrorException
  type: Class
  summary: <p>Exception thrown for an invalid response with <xref href="com.microsoft.azure.batch.protocol.models._batch_error" data-throw-if-not-resolved="false"></xref> information. </p>
  syntax:
    content: public class BatchErrorException extends RestException
- uid: com.microsoft.azure.batch.protocol.models._task_add_result
  parent: com.microsoft.azure.batch.protocol.models
  href: com.microsoft.azure.batch.protocol.models._task_add_result.yml
  name: TaskAddResult
  fullName: com.microsoft.azure.batch.protocol.models.TaskAddResult
  type: Class
  summary: <p>Result for a single task added as part of an add task collection operation. </p>
  syntax:
    content: public class TaskAddResult
- uid: com.microsoft.azure.batch.protocol.models._task_add_parameter
  parent: com.microsoft.azure.batch.protocol.models
  href: com.microsoft.azure.batch.protocol.models._task_add_parameter.yml
  name: TaskAddParameter
  fullName: com.microsoft.azure.batch.protocol.models.TaskAddParameter
  type: Class
  summary: <p>An <xref href="com.microsoft.azure._azure" data-throw-if-not-resolved="false"></xref> Batch task to add. </p>
  syntax:
    content: public class TaskAddParameter
- uid: com.microsoft.azure.batch._create_tasks_terminated_exception
  parent: com.microsoft.azure.batch
  href: com.microsoft.azure.batch._create_tasks_terminated_exception.yml
  name: CreateTasksTerminatedException
  fullName: com.microsoft.azure.batch.CreateTasksTerminatedException
  type: Class
  summary: >-
    <p>

      <xref href="com.microsoft.azure.batch._create_tasks_terminated_exception" data-throw-if-not-resolved="false"></xref>. </p>
  syntax: *o1
